docs/source/howto/query.rst

```diff
@@ -160,6 +159,59 @@ In case you want all calculation jobs with state ``finished`` or ``excepted``, y
         },
     )
 
+.. _how-to:query:filters:programatic:
+
+Programatic syntax for filters
```

Comment:
Think that programmatic is with two m's: https://www.merriam-webster.com/dictionary/programmatic